README.md

```diff
@@ -173,6 +173,26 @@ For [standalone installations](https://nix-community.github.io/home-manager/inde
   A: We primarily support the `unstable` branch, but try our best to support the current stable release.
   You can check if your stable release is currently supported at [status.nixos.org](https://status.nixos.org/)
 
+- Q: **"How do I fix the error: ... during evaluation because the option 'allow-import-from-derivation' is disabled"**\
+  A: Some ports require extra steps to configure those ports resulting in an [IFD](https://wiki.nixos.org/wiki/Import_From_Derivation).
```

Comment:
```suggestion
  A: Some ports need to read and/or manipulate remote resources, resulting in Nix performing [IFD](https://nix.dev/manual/nix/2.24/language/import-from-derivation).
```

Reply:
Just realized I recommended this URL change already...oops!

But in any case, it doesn't seem the wiki has improved on this topic and I don't think it will ever be much better than what's in the manual -- as that already provides explicit examples of functions that will caused IFD like import, readFile, etc. In the event it does, we can always replace it later; the manual link should be best for now IMO

Reply:
Used https://nix.dev/manual/nix/latest/language/import-from-derivation instead of hard coding the version.